x-pack/plugins/aiops/server/routes/log_rate_analysis.ts

```diff
@@ -63,23 +63,24 @@ const PROGRESS_STEP_GROUPING = 0.1;
 const PROGRESS_STEP_HISTOGRAMS = 0.1;
 const PROGRESS_STEP_HISTOGRAMS_GROUPS = 0.1;
 
-export const defineExplainLogRateSpikesRoute = (
+export const defineLogRateAnalysisRoute = (
```

Comment:
I don't think having the path passed in to a generic route registering function is correct way to do this.
In the future the `LOG_RATE_ANALYSIS` route will probably change and have its version bumped, and the old route should not change. However as it is now, the old route is the same as the new route and so it will also change versions.

I think the correct way to do this is to have two completely separate route registrations, and after some time we can delete the deprecated one.

Reply:
Thinking about this more, I'm now doubting myself.
It probably will be ok to leave as is, as users won't have this server side endpoint bookmarked. But that also makes me wonder if we need to keep the old one around at all?

In the future, just deleting deprecated code is easier than having to delete and change this function to remove the `path` variable. So maybe having a completely separate route registrations is cleaner. Even if it is identical code.

Reply:
As discussed offline, I removed the deprecated API endpoint in [a532e49](https://github.com/elastic/kibana/pull/161764/commits/a532e49c2e68865aca9d66763504868a3c092668). It was good as a dry run how this might play out in future serverless environments, but it's not necessary to keep the old endpoint around yet since it's internal and experimental.